dune-project

```diff
@@ -26,10 +26,11 @@
 
 (package
  (name lwt_ppx)
+ (version dev)
```

Comment:
I don't think this is needed.

Reply:
I added it mostly because then it's easier to remember to set the `version` field when doing a release. Otherwise I have to remember to add the `version` field which I'm likely to forget.

Probably should just write a release script, but in the meantime I'd rather have that.